test/samples/s3e/fitted_s3e/language_model_config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "embeddings_filename": "vectors.txt",
+  "hidden_size": 300,
+  "language": "German",
+  "name": "WordEmbedding_LM",
+  "vocab_filename": "vocab.txt",
+  "vocab_size": 4008
```

Comment:
unsure if this is really used, but it should be 113

Reply:
Fixed